src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -311,11 +311,16 @@ final class RiskProvider: RiskProviding {
 	private func _provideRiskResult(_ result: RiskProviderResult, to consumer: RiskConsumer?) {
 		#if DEBUG
```

Comment:
If I understand it right, this whole #if DEBUG section can be removed.
Because the _requestRiskLevel_Mock calls successOnTargetQueue and this calls _provideRiskResult.
What do you think?

Reply:
The DEBUG section is required for the inactive card.

Reply:
stripped obsolete code from the DEBUG section with commit 930d3cd194eb6786b356bb6f08e00c1dc9ba1afa

Reply:
Stripped obsolete code from the debug section with commit 930d3cd194eb6786b356bb6f08e00c1dc9ba1afa